ui/app/services/permissions.js

```diff
@@ -49,6 +49,12 @@ const API_PATHS = {
   settings: {
     customMessages: 'sys/config/ui/custom-messages',
   },
+  sync: {
+    destinations: 'sys/sync/destinations',
+    associations: 'sys/sync/associations',
+    config: 'sys/sync/config',
+    github: 'sys/sync/github-apps',
```

Comment:
where did these paths come from? are there any others we need to add here?

Reply:
From the API, I had a comment here before but comments get removed when I push changes. I'll add it to the PR Description instead.